### PR TITLE
inhibit applet - Allow power management inhibition on startup

### PR DESCRIPTION
**Pull request by claudiux**
Adds an option to the settings to enable inhibition of power management on start-up.

### CODE REVIEW

File: files/usr/share/cinnamon/applets/inhibit@cinnamon.org/applet.js
Comment (rcalixte) [inline files/usr/share/cinnamon/applets/inhibit@cinnamon.org/applet.js]: I've noticed on this and other pull requests that you're using tabs instead of spaces. The coding guidelines highlight [here](https://linuxmint-developer-guide.readthedocs.io/en/latest/guidelines.html#consistency) and [here](https://linuxmint-developer-guide.readthedocs.io/en/latest/guidelines.html#indentation) about why not to do so.

File: files/usr/share/cinnamon/applets/inhibit@cinnamon.org/settings-schema.json
Comment (rcalixte) [inline files/usr/share/cinnamon/applets/inhibit@cinnamon.org/settings-schema.json]: Same here except this is mixed spacing of what has been added.

**Review by rcalixte — CHANGES_REQUESTED** [review]

File: files/usr/share/cinnamon/applets/inhibit@cinnamon.org/applet.js
Comment (claudiux) [reply files/usr/share/cinnamon/applets/inhibit@cinnamon.org/applet.js]: Fixed.

File: files/usr/share/cinnamon/applets/inhibit@cinnamon.org/settings-schema.json
Comment (claudiux) [reply files/usr/share/cinnamon/applets/inhibit@cinnamon.org/settings-schema.json]: Also fixed.

**Comment by rcalixte** [comment]
Coding-wise, it works. One issue I see is that this has a conflict with #11744. I'll leave it up to others if this should be merged or not though.

**Comment by claudiux** [comment]
@rcalixte This applet already lets you disable notifications.

**Comment by claudiux** [comment]
@rcalixte I'm the one who didn't understand. In this PR, I'll take your changes into account.

**Comment by claudiux** [comment]
@rcalixte 
Your changes have been taken into account. That works well, good job!
I think there should be a specific icon when both are disabled.
Would you like me to do the same for notifications (disable at startup).

**Comment by rcalixte** [comment]
> Your changes have been taken into account. That works well, good job!

I don't think that's what should have been done. That PR is still waiting on review and final feedback. You should be avoiding a conflict with that not creating one. Your proposed change and the changes in the already open PR should be independent and as conflict-free as possible.



> I think there should be a specific icon when both are disabled.

There already is. I've been running this locally for months.


> Would you like me to do the same for notifications (ability to disable at startup)?

There is no need for this. Notifications are disabled via gsettings which are not reset on Cinnamon restarts or Cinnamon reboots.

**Review by rcalixte — CHANGES_REQUESTED** [review]
Please remove the other PR's proposed changes and try to avoid a conflict with that branch if possible.

**Comment by rcalixte** [comment]
@claudiux You have the ability to communicate with me directly. Instead of operating on your own in a state of confusion, I implore you to make use of that so that these situations can be avoided in the future. Two minutes of conversation would have prevented hours of wasted time. If this isn't agreeable, please let me know and we can find another solution that works for you. This current state of affairs is not good.

**Pull request closed without merging**
